src/SIM/AdaptiveSIM.C

```diff
@@ -309,7 +336,11 @@ bool AdaptiveSIM::adaptMesh (int iStep)
   {
     IFEM::cout <<"\nRefining by increasing solution space by "<< beta
                <<" percent."<< std::endl;
-    prm.errors = eNorm.getRow(eRow);
+    std::vector<DblIdx> errors(model.getPatch(1)->getNoNodes(1));
+    remapErrors(errors, model.getPatch(1), eNorm.getRow(eRow));
```

Comment:
doh. this is why i want review ;) i've been living in a mixed-only world. will fix.